app/charges/test_charges.py

```diff
@@ -464,4 +464,4 @@ def test_edit_charge_active_member_public_notbool(self):
 
         self.socketio.emit('edit_charge', user_data)
         received = self.socketio.get_received()
-        assert received[0]["args"][0] == Response.EditError
+        assert received[0]["args"][0] == Response.PermError
```

Comment:
I think it is clearer to a normal member who tries to edit a charge to see `Insufficient permissions to perform this action.` instead of `Couldn't edit charge.` To me, the second means that they should try to edit it again and could cause confusion.